.github/scripts/trymerge.py

```diff
@@ -973,6 +1034,9 @@ def merge(pr_num: int, repo: GitRepo,
     if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days:
         raise RuntimeError("This PR is too stale; the last push date was more than 3 days ago. Please rebase and try again.")
 
+    if land_checks:
+        commit = pr.create_land_time_check_branch(repo, 'viable/strict', force=force, comment_id=comment_id)
```

Comment:
What will happen if PR one wants to land depends on commit, which is already in `master`, but not yet in viable-strict?

Reply:
If your PR is based on a commit that's ahead of viable/strict, it'll just say it's up to date and the merge into viable strict won't do anything. We will be adding another option to let you choose whether or not to rebase onto master or viable/strict.